CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+## [Release 003] - 2019-08-29
+
 - Fixed a bug where missing "from" dates on Verify attributes caused a crash
 - stop user IP address appearing in Rollbar
```

Comment:
Can we correct the case on these lines? They should start with a capital letter.